FlaskApp/__init__.py

```diff
@@ -2,15 +2,23 @@
 import dotenv
 from flask_sqlalchemy import SQLAlchemy
 from sqlalchemy.sql import func
-from flask import Flask, render_template
+from flask import Flask, render_template, redirect, url_for
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, BooleanField, SubmitField
+from wtforms.validators import ValidationError, DataRequired, Email, EqualTo
+from flask_login import current_user, login_user, LoginManager
+from werkzeug.security import generate_password_hash, check_password_hash
+from flask import flash
```

Comment:
Consolidate 
"from flask import flash"
with the other flask imports

Also try to alphabetically order them